readme.md

```diff
@@ -53,24 +53,32 @@ You still might want to use `docker` and `docker-compose` for running SDK and DB
 
 `go run . db_migrate_up`
 
-**3. Start lbrytv API server**
+**3. Generate .rsa file**
+
+`ssh-keygen -t rsa -f token_privkey.rsa -m pem`
+
+**4. Start lbrytv API server**
 
 `go run .`
 
-**4. Clone [lbry-desktop](https://github.com/lbryio/lbry-desktop/) repo, if you don't have it**
+**5. Clone [lbry-desktop](https://github.com/lbryio/lbry-desktop/) repo, if you don't have it**
 
 ```
 cd ..
 git clone git@github.com:lbryio/lbry-desktop.git
 ```
 
-**5. Launch UI in lbry-desktop repo folder**
+**5. Install npm dependencies**
+
+`npm install`
```

Comment:
It's `yarn` as per [lbrytv-desktop readme suggestions](https://github.com/lbryio/lbry-desktop#prerequisites). That's why I think it's best to omit initialization steps for external repos from the readme.

Reply:
Good point. updated to remove that step. F